app/controllers/content/sections_controller.rb

```diff
@@ -109,8 +109,9 @@ def reorder
 
   private
 
-  def export_filename format
-    helpers.truncate(@section.title, length: 45, omission: '-', separator: ' ') + '.' + format
+  def export_filename format, annotations=false
```

Comment:
You can move this method and the section & casebook one into 1 method in the node model. 

Ex: 
`@section.export_filename('docx', @include_annotations)`

Reply:
Yeah, it seems goofy to have duplicate export logic in each of the three classes. Is it okay with you if I do not refactor that existing export logic as part of this PR? I just added a parameter to already-existing methods in the three classes. If you WOULD like that refactoring to happen at the same time, no problem, I'll close this and can add that later this week or early next week.

Reply:
That's fine!